data/systemd/insights-client-boot.service

```diff
@@ -0,0 +1,30 @@
+# This file is part of insights-client.
+#
+# Any changes made to this file will be overwritten during a software update. To
+# override a parameter in this file, create a drop-in file, typically located at
+# /etc/systemd/system/insights-client-boot.service.d/override.conf Put the desired
+# overrides in that file and reload systemd.
+#
+# For more information about systemd drop-in files, see systemd.unit(5).
+
+[Unit]
+Description=Run Insights Client at boot
+Documentation=man:insights-client(8)
+After=network.target
+ConditionPathExists=/etc/insights-client/.run_insights_client_next_boot
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/insights-client --retry 3
+Restart=no
+WatchdogSec=900
+CPUQuota=30%
+MemoryLimit=2G
+TasksMax=300
+BlockIOWeight=100
+ExecStartPost=-/bin/bash -c "echo 2G >/dev/null 2>&1 > /sys/fs/cgroup/memory/system.slice/insights-client.service/memory.memsw.limit_in_bytes"
+ExecStartPost=-/bin/bash -c "echo 1G >/dev/null 2>&1 > /sys/fs/cgroup/memory/system.slice/insights-client.service/memory.soft_limit_in_bytes"
+ExecStartPost=-rm /etc/insights-client/.run_insights_client_next_boot
```

Comment:
Why ignore return codes on these three commands?

For this final command,  adding the `--force` flag will cause rm to "ignore nonexistent files and arguments, never prompt."